tests/_data/models/UsersListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `3.0.x` branch?